### PR TITLE
[TIMOB-16909]: Fix anti-aliasing on views

### PR DESCRIPTION
**Pull request by hieupham007**
testing steps in JIRA

### CODE REVIEW

**Comment by ashcoding** [comment]
I tried this and it works nicely!

I have another scenario though which still has aliasing. :( 
File is in jira 'another_app.js' and resource image 'square.png'.

If I were to use:-
```
backgroundImage:'square.png',
```

There will still be aliasing.

File: android/titanium/src/java/org/appcelerator/titanium/view/TiBorderWrapperView.java [hunk 1]
```diff
@@ -111,17 +125,27 @@ private void updateBorderPath()
...
+		setAlphaAndColor();
+		canvas.drawPath(borderPath, paint);
+	}
+
+	private void setAlphaAndColor()
+	{
 		paint.setColor(color);
 		if (alpha > -1) {
 			paint.setAlpha(alpha);
     	}
```
Comment (ashcoding) [inline android/titanium/src/java/org/appcelerator/titanium/view/TiBorderWrapperView.java]: Minor thing: Curly bracket indentation is wrong.

**Comment by hieupham007** [comment]
Yes, this fix doesn't fix backgroundDrawable (clipping will cause aliasing for drawables). There's really no way for fixing drawables unless we create bitmaps from canvas and use PorterDuff xfer modes. That method will cause memory fragmentation and is not really viable.

This PR will fix backgroundColor for most cases. The aliasing is still there, but a lot less worse than before.

**Comment by ashcoding** [comment]
I came to a similar conclusion with respect to Drawables as what you mentioned. I guess there's no ideal way to solve that yet.

**Pull request merged**